### PR TITLE
feat: Add e2e user invite test suite (no-changelog)

### PR DESCRIPTION
**Pull request by alexgrozav**

### CODE REVIEW

File: cypress.config.js [hunk 1]
```diff
@@ -5,7 +5,7 @@ const BASE_URL = 'http://localhost:5678';
 
 module.exports = defineConfig({
 	retries: {
-		openMode: 1,
+		openMode: 0,
```
Comment (alexgrozav) [inline cypress.config.js:8]: For disabling retrying failed runs when developing.

File: cypress/e2e/14-sharing.cy.ts [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+	},
+	{
+		email: `${DEFAULT_USER_EMAIL}C`,
+		password: DEFAULT_USER_PASSWORD,
+		firstName: 'User',
+		lastName: 'C',
+	},
+];
+
+describe('Workflows', () => {
```
Comment (MiloradFilipovic) [inline cypress/e2e/14-sharing.cy.ts:37]: Nit: Probably should update description

**Review by MiloradFilipovic — DISMISSED** [review]
Looks good.

File: cypress/e2e/14-sharing.cy.ts (hunk 2 above)
Comment (alexgrozav) [reply cypress/e2e/14-sharing.cy.ts]: Good catch! Done.

**Review by MiloradFilipovic — DISMISSED** [review]
Re-approving after tests are fixed

**Review by MiloradFilipovic — APPROVED** [review]
Re-approve after merge

**Comment by janober** [comment]
Got released with `n8n@0.215.0`

**Pull request merged**